tsl/src/dist_util.c

```diff
@@ -101,7 +106,13 @@ dist_util_set_as_frontend()
 }
 
 bool
-dist_util_set_id(Datum dist_id)
+dist_util_set_id(Datum dist_uuid)
+{
+	return dist_util_set_id_with_metadata_check(dist_uuid, true);
+}
+
+static bool
+dist_util_set_id_with_metadata_check(Datum dist_id, bool check_metadata)
```

Comment:
It could involve more things. In the code before it included checking that `uuid` was non-null, and I think we can add more checks here if necessary.

Reply:
> It could involve more things. In the code before it included checking that `uuid` was non-null, and I think we can add more checks here if necessary.

When more checks are added, the name can be improved, like you do in this PR. If my suggestion correctly describes the current behaviour, then I suggest to use it instead of a name to cover unclear future.

Reply:
Renamed.